recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
Package from the defaults channel has `tclsh8.5` only
Conda-forge package has `tclsh8.5` and `tclsh`

You can give higher priority to conda-forge to use conda-forge packages. This has to be done by the user though.